tests/integrate_test/test_block_interval.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import argparse
+import time
+from txtool_utils import get_block_by_number, get_current_block_number, send_tx, get_receipt, get_metadata
+
+
+def main():
+    print(f'a) Test default block interval...')
+    current_height = get_current_block_number()
+    height = int(current_height, 16)
+
+    time.sleep(3)
+
+    next_height = str(hex(height + 1))
+    next_block = get_block_by_number(next_height)
+    assert next_block != None, "Default block interval is 3s"
+    metadata = get_metadata()
+    assert metadata['blockInterval'] == 3000
+
+    print(f'b) Send transaction to change block interval...')
+    # send tx to change the block interval from 3s to 6s
+    admin_privkey = '0x5f0258a4778057a8a7d97809bd209055b2fbafa654ce7d31ec7191066b9225e6'
+    system_contract_address = '0xffffffffffffffffffffffffffffffffff020000'
+    code = 'de7aa05d0000000000000000000000000000000000000000000000000000000000001770'  # setBlockInterval(6000)
+
+    opts = parse_arguments()
+    version = opts.version
+
+    tx_hash = send_tx(
+        admin_privkey, to=system_contract_address, code=code, version=version)
+    time.sleep(6)
+    receipt = get_receipt(tx_hash)
+    assert receipt['errorMessage'] == None, "Transaction execution failed"
+
+    print(f'c) Test changed block interval...')
+    current_height = get_current_block_number()
+    height = int(current_height, 16)
+    next_height = str(hex(height + 1))
+
+    time.sleep(3)
+
+    # block interval is 6s now, you could not get next block 3s after.
+    next_block = get_block_by_number(next_height)
+    assert next_block == None, "Block interval had changed to 6s"
```

Comment:
I don't quite understand why you are saying "Block interval had changed to 6s" here ?

Reply:
I send a tx to invoke `setBlockInterval`, and changed the default block interval from 3s to 6s.

Reply:
Yes, you have sent the transaction to `setBlockInterval`, but you can not say:
```
The blockchain has works as exactly 6s for its interval. 
```
from your test case, right?

As an option, you can count for more times, and every time, the interval is about 6s. And this can high probability say "It have really to change to 6s interval" .

Reply:
A stable test case about this is hard to design. 

In my view, the test is successfully as long as the default block interval changed after tx, i do not care how long the block interval you had set in tx.
 
In other words, I could not get a block after 3 seconds from now that shows block interval had changed. As for the how block interval changed, check it by `getMetaDta` is ok too.